src/Views/OperatingSystemView.vala

```diff
@@ -104,12 +104,20 @@ public class About.OperatingSystemView : Gtk.Grid {
         var bug_button = new Gtk.Button.with_label (_("Send Feedback"));
 
         Gtk.Button? update_button = null;
+        Gtk.Label? last_update_label = null;
         var appcenter_info = new GLib.DesktopAppInfo ("io.elementary.appcenter.desktop");
         if (appcenter_info != null) {
             update_button = new Gtk.Button.with_label (_("Check for Updates"));
             update_button.clicked.connect (() => {
                 appcenter_info.launch_action ("ShowUpdates", new GLib.AppLaunchContext ());
             });
+
+            var appcenter_settings = new GLib.Settings ("io.elementary.appcenter.settings");
+            var update_time = new GLib.DateTime.from_unix_utc (appcenter_settings.get_int64 ("last-refresh-time"));
```

Comment:
How would this behave if there was never an update done via AppCenter? Say, perhaps on a new installation?